### ISSUE TITLE
Store array property is not reactive when updated

### ISSUE THREAD

**Issue by ammezie** [ryangjchandler/spruce#46]
I have the following global store:

```js
Spruce.store('todo', {
    todos: [],
})
```

When I try adding new items to `todos`, but the update is not reactive.

Here's how I'm adding new item in the `todos` array:

```js
$store.todo.todos.push({
    title: 'First todo',
    complated: false,
})
```

**Comment by ryangjchandler** [comment]
@ammezie - Spruce's Proxy doesn't react too well to changes in arrays, so instead of calling `Array.push` you'll need to actually re-assign the value:

```js
$store.todo.todos = $store.todo.todos.concat({
    title: 'First todo',
    completed: false,
})
```

**Comment by ryangjchandler** [comment]
@ammezie - I've just released [v1.1.0](https://github.com/ryangjchandler/spruce/releases/tag/v1.1.0), please upgrade and `Array.push` will be supported.
